net/local/local_sendmsg.c

```diff
@@ -58,6 +58,18 @@
  ****************************************************************************/
 
 #ifdef CONFIG_NET_LOCAL_SCM
+static void local_freectl(FAR struct local_conn_s *conn, int count)
+{
+  FAR struct local_conn_s *peer = conn->lc_peer;
+
+  while (count-- > 0)
+    {
+      file_close(peer->lc_cfps[--peer->lc_cfpcount]);
+      kmm_free(peer->lc_cfps[peer->lc_cfpcount]);
+      peer->lc_cfps[peer->lc_cfpcount] = NULL;
+    }
```

Comment:
```suggestion
  while (count-- > 0)
    {
      file_close(conn->lc_cfps[--conn->lc_cfpcount]);
      kmm_free(conn->lc_cfps[conn->lc_cfpcount]);
      conn->lc_cfps[conn->lc_cfpcount] = NULL;
    }
```

Reply:
but we need free file handle in peer slot, not self.